jobs/ingest_cqc_care_directory.py

```diff
@@ -0,0 +1,232 @@
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import lit, collect_set, array, col, split, expr, when, length, struct, explode
+from pyspark.sql.types import StringType, IntegerType
+from utils import utils, cqc_care_directory_dictionaries
+import sys
+from schemas import cqc_location_schema, cqc_provider_schema
+import argparse
+
+
+def main(source, provider_destination, location_destination):
+    df = get_cqc_care_directory(source)
+
+    provider_df = convert_to_cqc_provider_api_format(df)
+    location_df = convert_to_cqc_location_api_format(df)
+
+    export_parquet_files(provider_df, provider_destination, location_df, location_destination)
+
+
+def get_cqc_care_directory(source):
+    print("Reading CSV from {source}")
```

Comment:
This is missing the f in front, so it will not do string interpolation.
```suggestion
    print(f"Reading CSV from {source}")
```